src/public/infernode/views/Manage/ManagePage.tsx

```diff
@@ -16,7 +17,7 @@ export default function ManagePage(): JSX.Element {
 
   return (
     <Stack direction="horizontal" gap={3}>
-      <ManageSidebar />
+      { captureSidebar ? <ManageSidebar /> : <></>}
```

Comment:
Should we use a new, separate manageSidebar state to flag this off, so we can turn on one sidebar when we've built it without turning on the other?